README.md

```diff
@@ -17,41 +15,105 @@ Its design facilitates research and accelerates prototyping in the intersection
 </div>
 
 ## Features
+- Reduced-coordinate physics engine for **fixed-base** and **floating-base** robots.
+- Multibody dynamics library for model-based control algorithms.
+- Fully Python-based, leveraging [jax][jax] following a functional programming paradigm.
+- Seamless execution on CPUs, GPUs, and TPUs.
+- Supports JIT compilation and automatic vectorization for high performance.
+- Compatible with SDF models and URDF (via [sdformat][sdformat] conversion).
+
+## Usage
+
+### Using Jaxsim as simulator
+
+
+```python
+import jax
+import jax.numpy as jnp
+import jaxsim.api as js
+import icub_models
+import pathlib
+
+# Load the iCub model
+model_path = icub_models.get_model_file("iCubGazeboV2_5")
+joints = ('torso_pitch', 'torso_roll', 'torso_yaw', 'l_shoulder_pitch',
+          'l_shoulder_roll', 'l_shoulder_yaw', 'l_elbow', 'r_shoulder_pitch',
+          'r_shoulder_roll', 'r_shoulder_yaw', 'r_elbow', 'l_hip_pitch',
+          'l_hip_roll', 'l_hip_yaw', 'l_knee', 'l_ankle_pitch', 'l_ankle_roll',
+          'r_hip_pitch', 'r_hip_roll', 'r_hip_yaw', 'r_knee', 'r_ankle_pitch',
+          'r_ankle_roll')
+ndof = len(joints)
+
+# Build and reduce the model
+model_description = pathlib.Path(model_path)
+full_model = js.model.JaxSimModel.build_from_model_description(
+    model_description=model_description, time_step=0.0001, is_urdf=True
+)
+model = js.model.reduce(model=full_model, considered_joints=joints)
+
+# Initialize data and simulation
+data = js.data.JaxSimModelData.zero(model=model).reset_base_position(
+    base_position=jnp.array([0.0, 0.0, 1.0])
+)
+T = jnp.arange(start=0, stop=1.0, step=model.time_step)
+tau = jnp.zeros(ndof)
+
+# Simulate
+for t in T:
+    data, _ = js.model.step(model=model, data=data, link_forces=None, joint_force_references=tau)
```

Comment:
What do you think about compressing the example code in a collapsible section?
I also removed an unused import and reformatted
```suggestion
<details>
<summary><b>Using JaxSim as a simulator</b></summary>

```python
import jax.numpy as jnp
import jaxsim.api as js
import icub_models
import pathlib

# Load the iCub model.
model_path = icub_models.get_model_file("iCubGazeboV2_5")

joints = ('torso_pitch', 'torso_roll', 'torso_yaw', 'l_shoulder_pitch',
          'l_shoulder_roll', 'l_shoulder_yaw', 'l_elbow', 'r_shoulder_pitch',
          'r_shoulder_roll', 'r_shoulder_yaw', 'r_elbow', 'l_hip_pitch',
          'l_hip_roll', 'l_hip_yaw', 'l_knee', 'l_ankle_pitch', 'l_ankle_roll',
          'r_hip_pitch', 'r_hip_roll', 'r_hip_yaw', 'r_knee', 'r_ankle_pitch',
          'r_ankle_roll')

# Build and reduce the model.
model_description = pathlib.Path(model_path)
full_model = js.model.JaxSimModel.build_from_model_description(
    model_description=model_description, time_step=0.001, is_urdf=True
)
model = js.model.reduce(model=full_model, considered_joints=joints)

# Initialize data and simulation.
data = js.data.JaxSimModelData.zero(model=model).reset_base_position(
    base_position=jnp.array([0.0, 0.0, 1.0])
)
T = jnp.arange(start=0, stop=1.0, step=model.time_step)
tau = jnp.zeros(model.dofs())

# Simulate.
for _t in T:
    data, _ = js.model.step(
        model=model, data=data, link_forces=None, joint_force_references=tau
    )
```

</details>

Reply:
SInce the example are quite short I would prefer to have them directly in the readme,  so that it is directly visible that it is possible to  use jaxsim within few lines of code, what do you think ?